hubcap/setup.py

```diff
@@ -0,0 +1,58 @@
+import collections
+import datetime
+import json
+import logging
+import os
+import re
+import requests
+
+from pathlib import Path
+
+NOW = int(datetime.datetime.now().timestamp())
+NOW_ISO = datetime.datetime.now(datetime.timezone.utc).astimezone().isoformat()
+
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-8s %(message)s',
+    level=logging.INFO,
+    datefmt='%Y-%m-%d %H:%M:%S'
+)
+
+PKG_EXCLUSION_OVERRIDES = {
+    'dbt-labs': ['bing-ads', 'recurly', 'shopify', 'purecloud', 'zendesk', 'stripe', 'outbrain', 'taboola',
+                 'quickbooks', 'bing-ads', 'taboola', 'heap', 'ecommerce', 'facebook-ads', 'zendesk', 'stripe',
+                 'purecloud'],
+    'beautypie': ['dbt-google-analytics-360', 'dbt-zendesk-support']
+}
+
+
+def build_config():
+    config = json.loads(os.environ['CONFIG'])
+    config['tracked_repos'] = {}
+
+    # necessary for as long as hubcap's git is disconnected from the github copy of the hub
+    response = requests.get('https://raw.githubusercontent.com/dbt-labs/hubcap/master/hub.json')
```

Comment:
I don't quite understand this. Why do we need to request the version of the `master` branch, rather than using the version available locally, and ensuring that Heroku is always running the latest version of the hubcap script (by pulling / checking out `master` earlier on)?

Reply:
Okay, haha, so this is the legacy code I referred to above. Since I've got the time today, I can edit this out. I'll do so in a few minutes and push a commit up so this will be resolved. 

For context, this was a necessary artifact from when the hubcap repo on Heroku was not the same as this hubcap repo. In fact, up until yesterday, the deployment involved manually copies of files, and to allow folks in the community to edit hub.json, this curl step had to be done separately. Now that we're auto deploying branches, we can take advantage of the local manifest (as we should).

Reply:
Patch (thankfully ended up being trivial). Things make more sense now.